serverless/service/types.py

```diff
@@ -48,7 +48,7 @@ def camel(self):
 
     @property
     def pascal(self):
-        return stringcase.pascalcase(self.identifier)
+        return stringcase.pascalcase(self.identifier.replace("-", "_"))
```

Comment:
I had an issue with this function turning `task-servcie` into `Taservice`

Reply:
there is a `snakecase` which will do what you need

Reply:
where you would like to use `snakecase`?